modules/assim.batch/R/minimize.GP.R

```diff
@@ -51,8 +51,8 @@ minimize.GP <- function(gp, rng, x0, splinefuns = NULL) {
   ybar <- tapply(gp$y, gp$x.id, mean)
   k    <- S22inv %*% (ybar - ey)
   
-  nlm(gpeval, x0, k = k, mu = ey, tau = tauwbar, psi = psibar,
-      x = gp$x.compact, rng = rng, splinefcns = splinefcns)
```

Comment:
buglet 1: This was passing NULL to gpeval because in minimize.GP the arg is `splinefuns` not `splinefcns`.